README.md

```diff
@@ -1,7 +1,7 @@
 ## Discovery
 
 ### Version
-> 1.5.8
+> 1.5.9
```

Comment:
Do u think this version is worth to be 1.6.0? I think it could be since we kind of introduce a new filter category. But it is up to you.

Reply:
Do we have any standards about when we bump a full version? I'm happy to go with whatever you suggest.